### PR TITLE
Add option to specify max-clients

### PR DESCRIPTION
**Pull request by sero583**
With this PR youre able to set how many clients ProxyPass will take.

### CODE REVIEW

**Comment by sero583** [comment]
Dont mind my mistakes, I was tired...😅

**Review by SupremeMortal — APPROVED** [review]

File: src/main/java/com/nukkitx/proxypass/Configuration.java [hunk 1]
```diff
@@ -27,6 +27,8 @@
     private boolean packetTesting = false;
     @JsonProperty("log-packets")
     private boolean loggingPackets = false;
+    @JsonProperty("max-clients")
+    private Integer maxClients = 0;
```
Comment (SupremeMortal) [inline src/main/java/com/nukkitx/proxypass/Configuration.java]: `int` instead of `Integer`

**Review by SupremeMortal — CHANGES_REQUESTED** [review]

File: src/main/java/com/nukkitx/proxypass/ProxyPass.java [hunk 2]
```diff
@@ -58,6 +58,7 @@
     private final AtomicBoolean running = new AtomicBoolean(true);
     private BedrockServer bedrockServer;
     private final Set<BedrockClient> clients = Collections.newSetFromMap(new ConcurrentHashMap<>());
+    private Integer maxClients = 0;
```
Comment (SupremeMortal) [inline src/main/java/com/nukkitx/proxypass/ProxyPass.java]: Missed another here

**Review by SupremeMortal — CHANGES_REQUESTED** [review]

File: src/main/java/com/nukkitx/proxypass/ProxyPass.java (hunk 2 above)
Comment (sero583) [reply src/main/java/com/nukkitx/proxypass/ProxyPass.java]: Thank you for the hint, changed it. What is the reasoning why you want to use primitves only?

**Review by SupremeMortal — APPROVED** [review]

**Pull request merged**